globocom/directional-navigation
# Fix - Correct handle focused bind problem

## Changes

The `_handleFocused` method was not doing `bind` properly. Changed to arrow function to make use of autobind.

## How to test

- Run the project on browser and make sure that there is no error in the Console.
